taxi_service/settings.py

```diff
@@ -24,7 +24,7 @@
              "-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
Code quality issue: DEBUG should not be set to False in your development settings. This can make it harder to debug issues during development. Consider creating a separate settings file for production where DEBUG can be set to False.